src/routes/safes/safes.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Inject, Injectable } from '@nestjs/common';
-import { max } from 'lodash';
+import max from 'lodash/max';
```

Comment:
To my knowledge, we can use Math.max with dates like this:
`Math.max(...dates)`
However, this returns a UNIX timestamp. To get a Date object, you can do:
`new Date(Math.max(...dates))`

That said, I believe using Lodash here makes the code more readable. I’ll leave it up to you to decide which approach to use.

Reply:
TypeScript doesn't like it with dates:

![image](https://github.com/user-attachments/assets/a47b2787-e787-4127-8279-962de9ee149c)

I've left it as is mainly due to readability.

Reply:
The TypeScript error occurs because `Math.max` does not return a `Date` object; we need to wrap it in `new Date`. Regarding readability, I agree with you—I’ll go ahead and resolve the thread.

Reply:
Not quite - we'd need to get the timestamp of each `Date` as `Math.max` only accepts `number[]`:

![image](https://github.com/user-attachments/assets/89d6009f-873a-4d73-9751-b226d5d23eee)